ci-operator/config/openshift/ovn-kubernetes/openshift-ovn-kubernetes-master.yaml

```diff
@@ -31,3 +31,7 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-e2e
+- as: e2e-ovn-hybrid-step-registry
+  steps:
+    cluster_profile: aws
+    workflow: origin-e2e-aws-ovn-hybrid
```

Comment:
Does the test that is part of this workflow check if OVN hybrid overlay is enabled?

Reply:
no, we have no testing around the hybrid overlay

Reply:
Please change the name to something else as we're not running e2e's with ovn-hybrid enabled

Reply:
@ravisantoshgudimetla that is exactly what @JacobTanenbaum is doing except there is no e2e that tests if ovn-hybrid got enabled.

Reply:
Right that's what we discussed on slack, so I am fine with the change.